packages/manager/modules/billing/src/autoRenew/actions/terminate-agora-service/controller.js

```diff
@@ -0,0 +1,33 @@
+import { TERMINATE_PATTERN } from './constants';
+
+export default class TerminateAgoraServiceController {
+  /* @ngInject */
+  constructor($translate, BillingTerminate, $q) {
+    this.TERMINATE_PATTERN = TERMINATE_PATTERN;
+    this.BillingTerminate = BillingTerminate;
+    this.$q = $q;
```

Comment:
You can remove this line as you are not using $q

Reply:
fixed !